cypress/integration/fileUpload.spec.ts

```diff
@@ -45,40 +45,37 @@ describe('File upload', () => {
       cy.get('div.btn-group > input[type=file]')
         .attachFile({ filePath: 'acme.png', mimeType: 'image/png' })
       cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
-        .should('have.text', `![](${ imageUrl })`)
+        .should('have.text', `![](${imageUrl})`)
     })
 
     it('via paste', () => {
       cy.fixture('acme.png')
         .then((image: string) => {
           const pasteEvent = {
             clipboardData: {
-              files: [Cypress.Blob.base64StringToBlob(image, 'image/png')]
-            }
+              files: [Cypress.Blob.base64StringToBlob(image, 'image/png')],
+              getData: (_: string) => ''
           }
-          cy.get('.CodeMirror-scroll')
-            .trigger('paste', pasteEvent)
-          cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
-            .should('have.text', `![](${ imageUrl })`)
-        })
+        }
+        cy.get('.CodeMirror-scroll').trigger('paste', pasteEvent)
+        cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
+          .should('have.text', `![](${imageUrl})`)
+      })
     })
 
     it('via drag and drop', () => {
-      cy.fixture('acme.png')
-        .then((image: string) => {
-          const dropEvent = {
-            dataTransfer: {
-              files: [Cypress.Blob.base64StringToBlob(image, 'image/png')],
-              effectAllowed: 'uninitialized'
-            }
+      cy.fixture('acme.png').then((image: string) => {
+        const dropEvent = {
+          dataTransfer: {
+            files: [Cypress.Blob.base64StringToBlob(image, 'image/png')],
+            effectAllowed: 'uninitialized'
           }
-          cy.get('.CodeMirror-scroll')
-            .trigger('dragenter', dropEvent)
-          cy.get('.CodeMirror-scroll')
-            .trigger('drop', dropEvent)
-          cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
-            .should('have.text', `![](${ imageUrl })`)
-        })
+        }
+        cy.get('.CodeMirror-scroll').trigger('dragenter', dropEvent)
+        cy.get('.CodeMirror-scroll').trigger('drop', dropEvent)
+        cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
+          .should('have.text', `![](${imageUrl})`)
```

Comment:
This seems to be malformatted, because these are chained function calls.

Reply:
Changed